outlines/welcome.md

```diff
@@ -0,0 +1,13 @@
+1. Welcome to the Meteor guide
+  1. This is a set of articles outlining opinions on best-practice application development in the Meteor framework
+  2. Nothing here is prescriptive (i.e. you can use the platform in many other ways), but this is how we think apps are best developed.
+  3. Hard APIs can be found at [docs] and community packages at [atmosphere].
+2. What is Meteor?
+  1. A curated set of JS client + server code for building web and mobile applications
```

Comment:
Some is curated, some is maintained by us. I'd say "curated" would imply that we didn't build any of it.